### PR TITLE
[sdl1]Change build method to vcpkg_*_make

### PR DESCRIPTION
**Pull request by JackBoosY**
- Use functions `vcpkg_*_make`.
- Correct dependency judgment conditions.

Related: https://github.com/microsoft/vcpkg/pull/8060#issuecomment-561039074 #8070

### CODE REVIEW

File: ports/sdl1/portfile.cmake [hunk 1]
```diff
@@ -53,62 +51,36 @@ if (VCPKG_TARGET_IS_WINDOWS)
         file(RENAME ${CURRENT_PACKAGES_DIR}/debug/lib/SDLmain.lib ${CURRENT_PACKAGES_DIR}/debug/lib/manual-link/SDLmaind.lib)
     endif()
 else()
-    find_program(autoreconf autoreconf)
-    if (NOT autoreconf OR NOT EXISTS "/usr/share/doc/libgles2/copyright")
-        message(FATAL_ERROR "autoreconf and libgles2-mesa-dev must be installed before libepoxy can build. Install them with \"apt-get dh-autoreconf libgles2-mesa-dev\".")
-    endif()
+    message("libgles2-mesa-dev must be installed before libepoxy can build. Install them with \"apt install libgles2-mesa-dev\".")
```
Comment (CJCombrink) [inline ports/sdl1/portfile.cmake]: Should probably be `sdl1` instead of `libepoxy`

**Comment by JackBoosY** [comment]
ogre is conflicts with ogre-next.

**Review by PhoebeHui — APPROVED** [review]

File: ports/sdl1/portfile.cmake [hunk 2]
```diff
@@ -53,62 +51,36 @@ if (VCPKG_TARGET_IS_WINDOWS)
         file(RENAME ${CURRENT_PACKAGES_DIR}/debug/lib/SDLmain.lib ${CURRENT_PACKAGES_DIR}/debug/lib/manual-link/SDLmaind.lib)
     endif()
 else()
-    find_program(autoreconf autoreconf)
-    if (NOT autoreconf OR NOT EXISTS "/usr/share/doc/libgles2/copyright")
-        message(FATAL_ERROR "autoreconf and libgles2-mesa-dev must be installed before libepoxy can build. Install them with \"apt-get dh-autoreconf libgles2-mesa-dev\".")
-    endif()
+    message("libgles2-mesa-dev must be installed before sdl1 can build. Install them with \"apt install libgles2-mesa-dev\".")
```
Comment (dan-shaw) [inline ports/sdl1/portfile.cmake]: them -> it

**Review by dan-shaw — APPROVED** [review]

**Pull request merged**